cfme/generic_objects/definition/ui.py

```diff
@@ -116,6 +118,7 @@ def instance_count(self):
     return int(view.summary("Relationships").get_text_of("Instances"))
 
 
+# -------------- navigator registers
 @navigator.register(GenericObjectDefinitionCollection)
```

Comment:
I don't see any comment for Navigation in code base.

Reply:
@sbulage I added these comments because usually the methods are not split into files like they are here. They are just to increase readability of the file, since I had some trouble organizing everything in my head when I initially looked at these files. Do you think I should just remove them?

 In my view, they only add to readability of the file.